README.md

```diff
@@ -4,4 +4,26 @@ An operator to install and configure [Shipwright](https://shipwright.io) on Kube
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to build, test, and submit contributions to the operator.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to build, test, and submit
+contributions to the operator.
+
+## Usage
+
+To deploy and manage instances of [Shipwright Build-Controller][build-controller], make sure this
+operator is up-and-running, and then create the following:
```

Comment:
I wonder if we should implement something where if the opeartor does not see a `ShipwrightBuild` instance in a certain amount of time, it creates a default instance.

Or maybe have a command line arg on the operator where it will create the default instance vs. wait for one.